### DISCUSSION TITLE
What is the process for properly updating all react-native-firebase packages

### DISCUSSION THREAD

**Discussion by brennansaul** [invertase/react-native-firebase#5891] [Q&A]
What is the process for properly updating all react-native-firebase packages?

I am currently using the following react-native-firebase packages and plan to use others:
- "@react-native-firebase/app": "^8.4.6",
- "@react-native-firebase/auth": "^9.3.1",
- "@react-native-firebase/database": "^7.5.12",
- "@react-native-firebase/storage": "^7.4.10",

as you can see the version In my package.json file are not the latest and I wanted to check to see what is the proper process for updating all packages. is there a CLI command to run or should I just increment the version number in the `package.json` to be the same as the latest version?

Thanks!

**Comment by mikehardy** [comment]
For versions before the switch to "One, True Version" (https://invertase.io/blog/react-native-firebase-versioning) you will have to look in each package's individual changelog for the breaking changes noted in each major version there.
With the release of v10 of all the packages there is a unified changelog that's much easier to follow. Then you just handle the breaking changes in each major version there.

I do my best in all breaking changes to provide enough info to answer the important questions immediately, right in the log entry: 1) am I affected? okay what do I do?

I do not recall any of the breaking changes being really difficult to deal with for the packages you are using. They *were* potentially breaking for some people so all deserving a semver major but not too hard to handle.

Good luck with your project!
